mantidproject/mantid
# dae time shift compensation

The changes allow to modify elastic line position in fix_ei mode

**To test:**
to test it properly one needs to be able to run reduction for LET. In this case you can set up time shift property and see that elastic line position changes. I did it and it works as requested. The changes to code are so simple (Added 4 lines of code, two identical in two places), that code review is sufficient for testing. 


Fixes #18698.

done on request of instrument scientist and is so minor so no need in release notes

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
